scripts/apps/templates/controllers/CreateTemplateController.ts

```diff
@@ -32,9 +34,23 @@ export function CreateTemplateController(
     this.types = templates.types;
     this.createTypes = _.filter(templates.types, (element) => element._id !== 'kill');
     this.save = save;
+    this.dateTimeFields = null;
 
     activate();
 
+    function itemData() {
+        const _item = JSON.parse(JSON.stringify(templates.pickItemData(item)));
+
+        self.dateTimeFields?.forEach((field) => {
+            if (_item.extra[field._id]) {
+                const initialOffset = field.custom_field_config.initial_offset_minutes;
```

Comment:
`initial_offset_minutes` config is an implementation detail of an extension. To maintain proper encapsulation, we can only implement the extensions API from core code(what extensions expose via `IExtensionActivationResult` interface). We can not put code in the core that is aware of implementation details of any particular extension.

For example, if you want custom fields from extensions to be able to set initial value when creating a template, you could add [an extension point similar to this](https://user-images.githubusercontent.com/2076953/144410684-446454d1-0379-42a4-afb4-945dc635eba3.png), and do it inside the extension then.